Binance Detect Moonings.py

```diff
@@ -140,9 +157,8 @@ def wait_for_price():
         # sleep for exactly the amount of time required
         time.sleep((timedelta(minutes=float(TIME_DIFFERENCE / RECHECK_INTERVAL)) - (datetime.now() - historical_prices[hsp_head]['BNB' + PAIR_WITH]['time'])).total_seconds())
 
-    print(f'Working...Session profit:{session_profit:.2f}% Est:${(QUANTITY * session_profit)/100:.2f}')
-
-    # retreive latest prices
+    print(f'Using {len(coins_bought)}/{TRADE_SLOTS} trade slots. Session profit: {session_profit:.2f}% - Est: {(QUANTITY * session_profit)/100:.{decimals()}f} {PAIR_WITH}')
```

Comment:
Honestly, the decimal work is only for better reporting, and right now we dont want to focus on how things look only the functionality, please leave as it was

Reply:
This is such a small enhancement and yet it really improves the display when trading crypto against crypto, I'd request a review of this please - users would really benefit from seeing actual figures and allowing 8 decimal places for crypto reporting lets them know that their bot is working.

For example, you trade with 0.006 ETH and make a 1% profit, thats 0.00006 ETH. At 2 decimal places it'd show 0.00.
Also, this would only increase for BTC as the trade value would be even lower using BTC min trade value of 0.0001.

Please reconsider 🙂